frontend/src/pages/Workflows/Workflow.js

```diff
@@ -35,17 +37,31 @@ const Workflow = (props) => {
         }}
       >
         {/* Button is disabled either if the user is not allowed to edit or the user is in "sort" mode */}
-        <Fab
-          disabled={createDisabled}
-          data-test="createWorkflowitem"
-          color="primary"
-          onClick={() => props.showCreateDialog()}
-          style={{
-            position: "relative"
-          }}
+        <Tooltip
+          title={
+            !allowedToCreateWorkflows
+              ? strings.common.no_permissions
```

Comment:
Can you move the logic  and make it more readable by using maybe nested if..else?

Reply:
@SamuelPull I changed logic to if/else